azure-mgmt-consumption/HISTORY.rst

```diff
@@ -3,19 +3,7 @@
 Release History
 ===============
 
-1.0.0 (2017-11-15)
-++++++++++++++++++
-
-**Features**
-
-- Featuring stable api GA version 2017-11-30
-- Supporting EA customers with azure consumption usage details
-
-**Breaking changes**
-
-- Removing support for calling usage_details.list() with 'invoice_id'. Will feature in future releases.
-
-0.1.0 (2017-05-18)
+0.1.0 (1970-01-01)
```

Comment:
Please do not override the ChangeLog, and even provide a 1.1.0 explaination of the reservations change

Reply:
I believe i did not make any changes manually to this file. t happens when i ran 
cookiecutter.exe -f gh:Azure/cookiecutter-azuresdk-pypackage
before creating pull request as mentioned here . 
https://github.com/Azure/azure-sdk-for-python/wiki/Generate-a-Python-SDK-from-a-Swagger-file
Am i missing something here